CHANGELOG.md

```diff
@@ -5,6 +5,85 @@
+  This is still an early experimental (alpha) feature. Do not recommended to be used production. To enable use "--featuregate=+pdata.useProtoPooling"
+- (Core) `pdata`: Optimize CopyTo messages to avoid any copy when same source and destination ([#13680](https://github.com/open-telemetry/opentelemetry-collector/pull/13680))
+- (Core) `receiverhelper`: New feature flag to make receiverhelper distinguish internal vs. downstream errors using new `otelcol_receiver_failed_x` and `otelcol_receiver_requests` metrics ([#12207](https://github.com/open-telemetry/opentelemetry-collector/issues/12207), [#12802](https://github.com/open-telemetry/opentelemetry-collector/pull/12802))
+  This is a breaking change for the semantics of the otelcol_receiver_refused_metric_points,  otelcol_receiver_refused_log_records and otelcol_receiver_refused_spans metrics.
+  These new metrics and semantics are enabled through the `receiverhelper.newReceiverMetrics` feature gate.
+- (Core) `debugexporter`: Add support for entity references in debug exporter output ([#13324](https://github.com/open-telemetry/opentelemetry-collector/issues/13324))
+- (Core) `pdata`: Fix unnecessary allocation of a new state when adding new values to pcommon.Map ([#13634](https://github.com/open-telemetry/opentelemetry-collector/pull/13634))
+- (Core) `service`: Implement refcounting for pipeline data owned memory. ([#13631](https://github.com/open-telemetry/opentelemetry-collector/issues/13631))
+  This feature is protected by `--featuregate=+pdata.useProtoPooling`.
+- (Core) `service`: Add a debug-level log message when a consumer returns an error. ([#13357](https://github.com/open-telemetry/opentelemetry-collector/issues/13357))
+- (Core) `xpdata`: Optimize xpdata/context for persistent queue when only one value for key ([#13636](https://github.com/open-telemetry/opentelemetry-collector/pull/13636))
+- (Core) `otlpreceiver`: Log the listening addresses of the receiver, rather than the configured endpoints. ([#13654](https://github.com/open-telemetry/opentelemetry-collector/issues/13654))
+- (Core) `pdata`: Use the newly added proto marshaler/unmarshaler for the official proto Marshaler/Unmarshaler ([#13637](https://github.com/open-telemetry/opentelemetry-collector/pull/13637))
+  If any problems observed with this consider to disable the featuregate `--feature-gates=-pdata.useCustomProtoEncoding`
+<!-- cspell:ignore MLKEM mlkem -->
+- (Core) `configtls`: Enable X25519MLKEM768 as per draft-ietf-tls-ecdhe-mlkem ([#13670](https://github.com/open-telemetry/opentelemetry-collector/issues/13670))
+- (Contrib) `k8sclusterreceiver`: Add optional experimental k8s.container.status.state metric ([#32457](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/32457), [#32457](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/32457))
```

Comment:
I was concerned our prepare-changelog script broke here with duplicate issue links, but it turns out it was a result of [upstream having the same issue listed twice](https://github.com/povilasv/opentelemetry-collector-contrib/blob/7c3964fe4cc0a98836a0812cbadb9df6031f7790/.chloggen/container-status.yaml#L11) in the changelog.